cache-config/t3c-apply/util/util.go

```diff
@@ -408,6 +408,14 @@ func doMkDir(name string, cfg config.Cfg, all bool) bool {
 					log.Errorf("unable to create the directory '%s': %v", name, err)
 					return false
 				}
+
+				if uid > -1 && gid > -1 {
```

Comment:
Negative uid and gids are valid on some *nix systems. Would you mind changing this to indicate with booleans or pointers? Maybe just change them to int*? ```if uid != nil && gid != nil { err = os.Chown(name, *uid, *gid)`

Reply:
changed as instructed